_api-reference/profile.md

```diff
@@ -753,4 +753,215 @@ Field | Description
+                    "time_in_nanos": 5366791,
+                    "reduce_time_in_nanos": 4637260,
+                    "max_slice_time_in_nanos": 4526680,
+                    "min_slice_time_in_nanos": 4414049,
+                    "avg_slice_time_in_nanos": 4487122,
+                    "slice_count": 3
+                  }
+                ]
+              }
+            ]
+          }
+        ],
+        "aggregations": [
+          {
+            "type": "NumericHistogramAggregator",
+            "description": "histo",
+            "time_in_nanos": 16454372,
+            "max_slice_time_in_nanos": 7342096,
+            "min_slice_time_in_nanos": 4413728,
+            "avg_slice_time_in_nanos": 5430066,
+            "breakdown": {
+              "min_build_leaf_collector": 4320259,
+              "build_aggregation_count": 3,
+              "post_collection": 9942,
+              "max_collect_count": 2,
+              "initialize_count": 3,
+              "reduce_count": 0,
+              "avg_collect": 146319,
+              "max_build_aggregation": 2826399,
+              "avg_collect_count": 1,
+              "max_build_leaf_collector": 4322299,
+              "min_build_leaf_collector_count": 1,
+              "build_aggregation": 3038635,
+              "min_initialize": 1057,
+              "max_reduce": 0,
+              "build_leaf_collector_count": 3,
+              "avg_reduce": 0,
+              "min_collect_count": 1,
+              "avg_build_leaf_collector_count": 1,
+              "avg_build_leaf_collector": 4321197,
+              "max_collect": 181266,
+              "reduce": 0,
+              "avg_build_aggregation": 954896,
+              "min_post_collection": 1236,
+              "max_initialize": 11603,
+              "max_post_collection": 5350,
+              "collect_count": 5,
+              "avg_post_collection": 2793,
+              "avg_initialize": 4860,
+              "post_collection_count": 3,
+              "build_leaf_collector": 4322299,
+              "min_collect": 78519,
+              "min_build_aggregation": 8543,
+              "initialize": 11971068,
+              "max_build_leaf_collector_count": 1,
+              "min_reduce": 0,
+              "collect": 181838
+            },
+            "debug": {
+              "total_buckets": 1
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+</details>
+
+### Modified or added response fields
+
+The following sections contain definitions of all modified or added response fields for concurrent segment search.
+
+#### The `query` array
+
+|Field	|Description	|
+|:---	|:---	|
+|`time_in_nanos`	|For concurrent segment search, `time_in_nanos` is the cumulative amount of time to run all methods across all slices, in nanoseconds. This is not equivalent to the actual amount of time the query took to run because it does not take into account that multiple slices can run the methods in parallel.	|
```

Comment:
```suggestion
|`time_in_nanos`	|For concurrent segment search, `time_in_nanos` is the cumulative amount of time taken to run all methods across all slices, in nanoseconds. This is not equivalent to the actual amount of time the query took to run because it does not take into account that multiple slices can run the methods in parallel.	|
```